src/05-led-roulette/flash-it.md

```diff
@@ -102,7 +102,9 @@ Remote debugging using :3333
 0x00000000 in ?? ()
 ```
 
-**NOTE**: If you are getting errors like `undefined debug reason 7 - target needs reset` on the OpenOCD GDB server, then you may need to try using `arm-none-eabi-gdb` instead of the `gdb` command, as described above.
+**NOTE**: If you are getting errors like `undefined debug reason 7 - target needs reset`, you can try running `monitor reset halt` as described [here](https://stackoverflow.com/questions/38994596/reason-7-target-needs-reset-unreliable-debugging-setup).
+
+**NOTE**: If the debugger is still not connecting to the OpenOCD server, then you may need to try using `arm-none-eabi-gdb` instead of the `gdb` command, as described above.
```

Comment:
These changes are in response to this comment: https://github.com/rust-embedded/discovery/commit/ffeabc63d758085734bb7983ac3848b973c5f2ea#r34883585

My original comment incorrectly conflated the `undefined debug reason 7` output with my issue where I was using `gdb` instead of `arm-none-eabi-gdb`.  This update clarifies the issues that may arise, and addresses them separately.